server/tables/tables.go

```diff
@@ -996,6 +996,30 @@ func isStrictModeEnabled(db *gorm.DB) (bool, error) {
 	return isStrictModeEnabled, nil
 }
 
+// RegisterLogSQLCallback adds a callback to the db that appends all SQL executed to a slice of strings
+func RegisterLogSQLCallback(db *gorm.DB, s *[]string, excludeQueries bool) error {
```

Comment:
can we move these two functions to some other libraries? for example, gormutil.go?

it feels weird to me that clickhouse.go is depended on tables.go (which is mysql table definition)